font-icons/examples-icons.md

```diff
@@ -0,0 +1,95 @@
+| <span class="icon-products"></span> products | __\&#xe8a7;__ | __\ue8a7__ | <span class="icon-customer"></span> customer | __\&#xe8a8;__ | __\ue8a8__ | <span class="icon-export"></span> export | __\&#xe8a9;__ | __\ue8a9__ |
+| <span class="icon-info-outlines"></span> info-outlines | __\&#xe8ac;__ | __\ue8ac__ | <span class="icon-contract"></span> contract | __\&#xe8dd;__ | __\ue8dd__ | <span class="icon-enlarge"></span> enlarge | __\&#xe8de;__ | __\ue8de__ |
+| <span class="icon-translate"></span> translate | __\&#xe8df;__ | __\ue8df__ | <span class="icon-emoji"></span> emoji | __\&#xe900;__ | __\ue900__ | <span class="icon-brightness"></span> brightness | __\&#xe901;__ __\ue901__ |
+| <span class="icon-flip-vertical"></span> flip-vertical | __\&#xe902;__ | __\ue902__ | <span class="icon-flip-horizontal"></span> flip-horizontal | __\&#xe903;__ | __\ue903__ | <span class="icon-rotate-cw"></span> rotate-cw | __\&#xe904;__ | __\ue904__ |
+| <span class="icon-rotate-ccw"></span> rotate-ccw | __\&#xe905;__ | __\ue905__ | <span class="icon-crop"></span> crop | __\&#xe906;__ | __\ue906__ | <span class="icon-hue"></span> hue | __\&#xe907;__ | __\ue907__ |
+| <span class="icon-link-external"></span> link-external | __\&#xf08e;__ | __\uf08e__ | <span class="icon-plus-squared"></span> plus-squared | __\&#xf0fe;__ | __\uf0fe__ | <span class="icon-angle-left"></span> angle-left | __\&#xf104;__ | __\uf104__ |
+| <span class="icon-angle-right"></span> angle-right | __\&#xf105;__ | __\uf105__ | <span class="icon-angle-up"></span> angle-up | __\&#xf106;__ | __\uf106__ | <span class="icon-angle-down"></span> angle-down | __\&#xf107;__ | __\uf107__ |
+| <span class="icon-spinner"></span> spinner | __\&#xf110;__ | __\uf110__ | <span class="icon-arrow-circled-left"></span> arrow-circled-left | __\&#xf137;__ | __\uf137__ | <span class="icon-arrow-circled-right"></span> arrow-circled-right | __\&#xf138;__ | __\uf138__ |
+| <span class="icon-minus-squared"></span> minus-squared | __\&#xf146;__ | __\uf146__ | <span class="icon-minus-squared-alt"></span> minus-squared-alt | __\&#xf147;__ | __\uf147__ | <span class="icon-plus-squared-alt"></span> plus-squared-alt | __\&#xf196;__ | __\uf196__ |
+
+
+>important You need to set the Telerik example font icon code on the concrete property to visualize the icon. 
+
+## Examples
+
+This example shows how to add the font icons code to a Label. For this case you need to add the icon code to the `Label.Text` property and set the `FontFamily`:
+
+```XAML
+<Label Text="&#xe800;" FontFamily="TelerikFontExamples"/>
+```
+```C#
+var label = new Label
+{
+    Text = "\ue800",
+    FontFamily = "TelerikFontExamples"
+};
+```
+
+>tip The `FontFamily` name is the name of the font registered in the `MauiProgram.cs` file
+
+## See Also
+
+- [Controls Icons]({%slug telerik-controls-icons%})
```

Comment:
Maybe add a link to the Overview and the github repo with the downloadable files.